docs/source/tutorials/tutorials.rst

```diff
@@ -3,10 +3,24 @@
 Tutorials
 =========
 
-Here go the first examples needed to start with Rascal.  
+Here you will find some tutorial examples for getting started with librascal,
+both as a new user of the library and for getting started developing a new
+representation.
 
 
+Getting started
+~~~~~~~~~~~~~~~
+
+.. toctree::
+    :maxdepth: 1
+
+    TutorialEn
+    SOAP_CSD500.ipynb
```

Comment:
What do you think of putting the explanations of `TutorialEn` also in an ipynb? I would use doxygen tutorials only for explanation referring code inside the python library code as we do it for c++

Reply:
Not sure I understand the second sentence. We can rework the tutorials as needed in the new `feat/tutorials` branch.

Reply:
And don't forget that static results/images that you'd want to be displayed in the web interface are currently at odds with out ipython notebook stripout policy. Though I think you can get `nbsphinx` to automatically run all notebook cells before converting them to html, even if I'm not sure how much of a pain it is to make sure it's using the correct python environment...